app/components/CopyDocButton.vue

```diff
@@ -0,0 +1,103 @@
+<script setup lang="ts">
+import type { ContentCollectionItem } from '@nuxt/content';
+import { useClipboard } from '@vueuse/core';
+
+const props = defineProps<{
+	page: ContentCollectionItem;
+}>();
+
+const isCopied = ref(false);
+
+const items = [
+	{
+		label: 'Copy Page',
+		description: 'Copy page as Markdown for LLMs',
+		icon: 'material-symbols:content-copy-outline',
+		onSelect: () => copyPage(),
+	},
+	{
+		label: 'View as Markdown',
+		description: 'View this page as plain text',
+		icon: 'material-symbols:markdown-outline',
+		onSelect: () => {
+			return navigateTo(`https://github.com/directus/docs/raw/refs/heads/main/content/${props.page?.stem}.md`, {
+				open: {
+					target: '_blank',
+				},
+			});
+		},
+	},
+	{
+		label: 'Open in ChatGPT',
+		icon: 'i-simple-icons:openai',
+		target: '_blank',
+		onSelect() {
+			window.open(`https://chatgpt.com/?hints=search&q=${encodeURIComponent(`Read ${window.location.href} so I can ask questions about it.`)}`, '_blank');
+		},
+	},
+	{
+		label: 'Open in Claude',
+		icon: 'i-simple-icons:anthropic',
+		target: '_blank',
+		onSelect() {
+			window.open(`https://claude.ai/new?q=${encodeURIComponent(`Read ${window.location.href} so I can ask questions about it.`)}`, '_blank');
+		},
+	},
+];
+
+async function copyPage() {
+	try {
+		const value = (props.page?.rawbody ?? '').replace(/\\n/g, '\n');
```

Comment:
When copying the markdown of a page, it does not resolve partials...wondering if it should? For instance, copying start has this line in it `:partial{content="license"}` instead of the license partial!

Reply:
Ok! We can just mark that as an improvement for later I think.